### PR TITLE
feat: add cfn for daemon infra

### PR DESCRIPTION
**Pull request by rutujasurve94**
`INFRASTRUCTURE`:
`Client`:
git and jina installed
`Encoder`:
Torch installed only in the encoder
GPU available in the encoder
Jinad installed and running with logs outputed somewhere that we can at least see or tail -f
`Indexer`:
Large disk space
Docker daemon running and docker sdk installed
Large disk space
Redis installed and service started and redis python dependency installed
`Ranker`:
Rename to Ranker-Redis because it will be used differently depending on the example
Large disk space (I guess we can start not so big but this has to be easily customizable)
Redis installed and service started and redis python dependency installed
`Crafter`:
No special setting would be needed
Jinad installed and running with logs outputed somewhere that we can at least see or tail -f

- `VPC`
- `Security group` with `Ingress/outgress`
- 1 instance per `pod`
- 1 instance fro `flow`

### CODE REVIEW

File: stress_test_trigger/cloud_ymls/stress-test-daemon.yml [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
```
Comment (rutujasurve94) [inline stress_test_trigger/cloud_ymls/stress-test-daemon.yml:1]: key-pair attachment to all instances for login

File: stress_test_trigger/cloud_ymls/stress-test-daemon.yml [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Template to run stress-test for Jina on AWS
+Parameters:
+  EC2ImageIdParam:
+    Type: String
+    Default: ami-00831fc7c1e3ddc60
+    Description: Enter Image ID. Default is for Ubuntu 20.04
+  EC2InstanceTypeParam:
```
Comment (rutujasurve94) [inline stress_test_trigger/cloud_ymls/stress-test-daemon.yml:9]: Is only 1 instance ok for both chunk and doc index?
Make IP of all pods available as IP1, IP2, etc.

```yaml:
port_expose:
host:
``` in each of the pods

jina, jinad, fluentd on `cfn-init` using script
and mounting of volumes

Comment (rutujasurve94) [reply stress_test_trigger/cloud_ymls/stress-test-daemon.yml:9]: - `PEM` file on console for logging in : key -> val

- Name for each of the instances

- Output of the infra creation of instances should be the `EIP` of the instances.

Comment (rutujasurve94) [reply stress_test_trigger/cloud_ymls/stress-test-daemon.yml]: Write `README` for infra.

**Review by deepankarm — APPROVED** [review]

**Pull request merged**